matchms/filtering/remove_peaks_outside_top_k.py

```diff
@@ -0,0 +1,44 @@
+import numpy
+from ..Spikes import Spikes
+from ..typing import SpectrumType
+
+
+def remove_peaks_outside_top_k(spectrum_in: SpectrumType, k: int = 6,
+                               mz_window: float = 50) -> SpectrumType:
+
+    """Remove all peaks which are not within *mz_window* of at least one
+       of the *k* highest intensity peaks of the spectrum.
+
+    Parameters
+    ----------
+    spectrum
```

Comment:
Match docstring with function spec
```suggestion
    spectrum_in
```